<launch>
    <node name="irap_serial_proxy_node" pkg="irap_serial_proxy" type="irap_serial_proxy_node" output="screen">
        <param name="device" type="str" value="/dev/ttyACM0"/>
        <param name="baudrate" type="int" value="115200"/>
        <param name="sending_rate" type="int" value="100"/>
        <param name="pub_tf" type="bool" value="true"/>
        <param name="pub_odom" type="bool" value="true"/> 
    </node>

</launch>